apps/root/src/frame/index.tsx

```diff
@@ -139,6 +139,7 @@ const AppFrame = ({ config: { wagmiClient }, initialChain }: AppFrameProps) => {
                               ))}
                               <Route path="/history" element={<History />} />
                               <Route path="/earn" element={<Earn />} />
+                              <Route path="/earn/:assetTokenId?/:rewardTokenId?" element={<Earn />} />
```

Comment:
strategies will always be tied to a chainId, so it could be a parameter, to aslo conform to the other routes that we have, and assetId and rewardId is only the token address